### DISCUSSION TITLE
operations.ts file

### DISCUSSION THREAD

**Discussion by kathirsvn** [stargate/stargate-mongoose#58] [General]
@vkarpov15 @kidrecursive 
I see a file called [operations.ts](https://github.com/stargate/stargate-mongoose/blob/main/src/collections/operations.ts) with an empty definition called `handleDocumentUpdateOperators`. Could you please explain the purpose of this file, so I can work on it if required?

cc: @jeffreyscarpenter

**Comment by kidrecursive** [comment]
Heya, this was for handling update operators on the client, like `$inc`, it's no longer needed.

**Reply by kathirsvn** [reply]
Got it, thanks @kidrecursive! I'll remove that file.
